packages/api/src/util/getMetadata.ts

```diff
@@ -0,0 +1,50 @@
+import { getProvider } from '@cennznet/api/util/getProvider';
+import { TypeRegistry } from '@polkadot/types/create';
+import { RpcCore } from '@polkadot/rpc-core';
+import { MetadataVersioned } from '@polkadot/metadata/MetadataVersioned';
+import { Metadata } from '@polkadot/metadata';
+import { ProviderInterface } from '@polkadot/rpc-provider/types';
+import { WsProvider } from '@polkadot/rpc-provider';
+
+const KEEP = ['genericasset', 'cennzx'];
+const MUST = ['system', 'timestamp', 'transactionpayment'];
+export async function getMetadata(
```

Comment:
can you doc comment this please

what's the difference between keep and must?

I think this can create some issue if user wants to add a module `modules: ['staking']` it will break functionality for genericasset + cennzx which worked before implicitly.